src/Sulu/Bundle/MediaBundle/Resources/js/components/ImageFocusPoint/FocusPoint.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import {Icon} from 'sulu-admin-bundle/components';
+import type {Point} from './types';
+import focusPointStyles from './focusPoint.scss';
+
+const ARROW_UP_ICON = 'arrow-up';
+
+type Props = {
+    size: number,
+    value: Point,
+    active: boolean,
+    onClick?: (value: Point) => void,
+    showArrow: boolean,
+    arrowDirection: number,
+};
+
+export default class FocusPoint extends React.PureComponent<Props> {
```

Comment:
I think that this name is a little bit confusing... Why not calling it something like `ImageFocusPointCell`? Optional I would still extract something like a `SelectionGrid` 😉

Reply:
It's not that easy to create a generic grid which can be used for this component. I wouldn't do that now when there is no need for that.